rust/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
```

Comment:
We need a build flag since tokio tracing instrumentation is only available in their unstable release